.travis.yml

```diff
@@ -14,15 +14,26 @@ stages:
 jobs:
   include:
     - stage: 'Lint markdown files'
-      language: ruby
-      rvm: 2.4.1
       os: linux
+      language: generic
       install: gem install awesome_bot
       script:
         - find . -type f -name '*.md' -exec awesome_bot {} \;
       notifications:
         email: false
 
+    - stage: 'Lint markdown files'
+      os: linux
+      language: generic
+      before_install: skip
+      install:
+        - npm i -g markdown-spellcheck
+      before_script:
+        - wget --quiet https://raw.githubusercontent.com/optimizely/mdspell-config/master/.spelling
```

Comment:
I thought we wanted to keep that file locally. Are we going to make a change to this global file all the time?

Reply:
we want to use global so we dont have to maintain redundant entries for all sdk repos.

eg.

optimizely is special dictionary

should be for every sdk.. lets just put it in one place